doc/source/whatsnew/v0.21.0.txt

```diff
@@ -116,6 +116,8 @@ Other API Changes
 Deprecations
 ~~~~~~~~~~~~
 - :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
+- :method:`Series.argmax` has been deprecated in favor of :method:`Series.idxmax` (:issue:`16830`)
```

Comment:
@jreback Note that in 'theory', `:method:` is more correct here (as idxmax is a method, not a function), but in practice both work, so it is not that important